src/main/java/core/basesyntax/Zoo.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import core.basesyntax.animal.Animal;
+import core.basesyntax.animal.Bird;
+import core.basesyntax.animal.Fish;
+import core.basesyntax.animal.Flyable;
+import core.basesyntax.animal.LandAnimal;
+import core.basesyntax.animal.Swimmable;
+
+public class Zoo {
+    /**
+     * Вольер для сухопутных животных
+     */
+    private final Aviary<LandAnimal> landAnimalAviary = new Aviary<>();
+    /**
+     * Вольер для птиц
+     */
+    private final Aviary<Bird> birdAviary = new Aviary<>();
+    /**
+     * Вольер для рыб (аквариум)
+     */
+    private final Aviary<Fish> fishAviary = new Aviary<>();
```

Comment:
try to use Interface in generic instead of specific class

Reply:
fixed